pages/grids/index.js

```diff
@@ -1,6 +1,7 @@
 /* global window:false */
 
-import '@babel/polyfill'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
I'm not sure about these `import/no-extraneous-dependencies` but I didn't see any warnings/errors